src/api/public/apidocs/paths/source_project_name_package_name_meta.yaml

```diff
@@ -6,6 +6,13 @@ get:
   parameters:
     - $ref: '../components/parameters/project_name.yaml'
     - $ref: '../components/parameters/package_name.yaml'
+    - in: query
+      name: deleted
+      schema:
+        type: string
+        enum:
+          - null
```

Comment:
```suggestion
        enum:
          - null
```
This translates into passing the string `'null'` instead of passing the parameter without a value. I would prefer not to define the `enum` in this case.

Reply:
```suggestion
      allowEmptyValue: true
```
Forget my previous proposal: adding `allowEmptyValue: true` will make it. :sweat_smile:

![Peek 2023-07-03 16-12-swagger-ui-deleted-parameter-allow-empty-value](https://github.com/openSUSE/open-build-service/assets/24919/f00d9a6b-9390-4090-9576-b90b197cbd17)